doc/themes/the-things-stack/layouts/partials/proto/glossary-term.html

```diff
@@ -0,0 +1,18 @@
+<dl class='glossary'>
+	<dt id="{{.term | urlize}}">
```

Comment:
Shouldn't the term, abbr and alts each be rendered into their own `dt`, like in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#Multiple_terms_single_description

Reply:
yep